src/types.ts

```diff
@@ -179,6 +180,18 @@ export interface Instance {
     signer: ethers.Signer
   ): Promise<ethers.ContractTransaction>;
 
+  /**
```

Comment:
nit: the function name is pretty self explanatory but a comment here for consistency would be nice

Reply:
🧛